airflow/ui/src/pages/TaskInstance/Logs.tsx

```diff
@@ -110,6 +136,26 @@ export const Logs = () => {
       <Code overflow="auto" py={3} textWrap={wrap ? "pre" : "nowrap"}>
         <VStack alignItems="flex-start">{data.parsedLogs}</VStack>
       </Code>
+      <Dialog.Root
+        onOpenChange={onOpenChange}
+        open={fullscreen}
+        scrollBehavior="inside"
+        size="full"
+      >
+        <Dialog.Content backdrop>
+          <Dialog.Header>
+            <Heading size="xl">Log</Heading>
+          </Dialog.Header>
+
+          <Dialog.CloseTrigger />
+
+          <Dialog.Body>
```

Comment:
We should keep the try selector and wrap/unwrap button in the modal

Reply:
Thanks, I suppose the entire content from can become a component like `<LogComponent>` and then show `FullScreen` based on the present inside the dialog or not like below. Are there any other ways to do this idiomatic in react without a separate component?

```html
<Box>
    <LogComponent showFullScreenButton />
    <Dialog>
        <LogComponent />
    </Dialog>
</Box>
```

Reply:
I tried copy pasting code to try this out. IMO it still takes up some space with tries, I would prefer selecting the log and then clicking on the fullscreen to utilise the screen space fully.

Full screen with task tries and wrap

![image](https://github.com/user-attachments/assets/6095753d-fa80-42cd-932d-c209bf303820)

Fullscreen without dialog header

![image](https://github.com/user-attachments/assets/11214b5c-0db1-4f2d-8a75-8e38e6450d70)

Reply:
+1 for keeping the header. It's taking really few real estate in the page and allowing to navigate without having to close and then open again the full screen mode. 

That's also consistant with how we build the graph,grid, etc... views. (there's a header). Also maybe  the Task id too would be helpful. (for the grid graph etc... we have the dag_id). I can easily imagine someone losing track of what logs he is looking at after a while reading in the full screen page. Let's imagine we have multiple opened tabs like this at the same time, it's difficult to navigate and remember about.

Reply:
Thanks @bbovenzi and @pierrejeambrun for the feedback. It also makes it consistent with other pages if this is the intended design. I will wait for @bbovenzi comment to see if there is any way to do this in react since without a component or if a component is the best way.